data/summary.yaml

```diff
@@ -148,7 +148,7 @@ Docker GitHub Copilot:
   availability: Early Access
 Docker Model Runner:
   availability: Beta
-  requires: Docker Desktop 4.40 and later
+  requires: Docker Engine 28.2 or Docker Desktop (Windows) 4.41+ or Docker Desktop (MacOS) 4.40+
```

Comment:
While the `docker-model-plugin` packages will ship together with the 28.2 engine release, I don't think there's strictly a version-requirement on the engine that's used for the CLI plugin to work (unless the plugin use a API-version constraint). So perhaps we should not include it as a _constraint_ here.


(i.e., if I have the docker package repository configured, I can `apt-get install docker-model-plugin` to add the CLI plugin to an existing install, even without updating the docker engine (and CLI))